zetaclient/chains/evm/constant.go

```diff
@@ -3,12 +3,13 @@ package evm
 import "time"
 
 const (
-	// ZetaBlockTime is the block time of the Zeta network
-	ZetaBlockTime = 6500 * time.Millisecond
-
 	// OutboundInclusionTimeout is the timeout for waiting for an outbound to be included in a block
 	OutboundInclusionTimeout = 20 * time.Minute
 
+	// ReorgProtectBlockCount is confirmations count to protect against reorg
+	// Short 1~2 block reorgs could happen often on Ethereum due to network congestion or block production race conditions
+	ReorgProtectBlockCount = 2
```

Comment:
Why is ReorgProtectBlockCount only 2? We use 32(or 16?)  confirmations for observations so how come this value is so much lower?

Reply:
Good question.

The number `2` is used by tracker reporter to decide whether or not the tx hash is a good candidate to be added to outbound tracker during that `20-minute` period. The tracker list can hold at most `5` candidates. Each of candidate will eventually be checked again by a final stringent verification process which employes `32` confirmations.